mteb/tasks/Classification/guj/GujaratiNewsClassification.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class GujaratiNewsClassification(AbsTaskClassification):
```

Comment:
This all seem to be the same underlying dataset so I would rephrase it as a Multilingual task.

Reply:
I agree with @KennethEnevoldsen, it would be better to have only one task that is Multilingual. Since you created the datasets, mayeb you can put them all into one repo and add languages subsets as in [mlsum](https://huggingface.co/datasets/mlsum). Also, do you have any references for this dataset ?

Reply:
Thanks a lot for the review. I will refactor this to be a multilingual task.